content/handbook/support/workflows/dedicated_cmoc.md

```
@@ -0,0 +1,204 @@
+## コミュニケーションのモード
+
+GDCMOC の役割には、それぞれ異なる目的を持ち、異なるツールを使用する 2 種類の顧客コミュニケーションが含まれます。ページされたとき、GitLab Dedicated SRE は、顧客に通知する必要があるのか、または情報を収集する必要があるのかに応じて、どの方法が必要かをアドバイスできます。GDCMOC にトラブルシューティングの責任を遂行することは期待されていません。
+
+**モード 1: Switchboard 通知**
+
+- インシデントステータスや、1 つ以上の顧客環境に影響を与える緊急メンテナンス計画について顧客に通知するための、一方向のブロードキャストコミュニケーション
+- [Switchboard](dedicated_switchboard.md) で事前承認されたテンプレートを使用して通知を作成する必要がある
+- 通知は、テナントの **[運用メールアドレス](https://docs.gitlab.com/administration/dedicated/tenant_overview/#contact-information)** リストと、メール通知が有効な Switchboard ユーザーに送信される
+- このコミュニケーションモードは、インシデント時のスケーラブルでコンプライアンスに準拠した顧客コミュニケーションを提供するため、テナントごとに個別のサポートチケットを作成するという以前の手動プロセスを置き換えます。[STM#6768 を参照](https://gitlab.com/gitlab-com/support/support-team-meta/-/issues/6768)
+- GitLab Unfiltered YouTube アカウントを使用したこの機能の [内部デモ](https://www.youtube.com/watch?v=e2ZRD8csjow) を視聴
+- → [Switchboard を使用した通知の送信](#sending-notifications-using-switchboard) に従う
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)